README.md

```diff
@@ -37,33 +37,69 @@ documentation](https://spladder.readthedocs.io/en/latest/).
+- `--batch-size`: Default = 1000. Number of genes to be processed in each batch. If bigger batches are selected, the program will be faster, but it will require more memory.
+- `--pickle-samples`: Default = []. List of samples to be pickled. Needed if `--use-mut-pickle` is set to True. It will create intermediate files containing mutation information of the selected samples. When mutation information is needed, it will be directly loaded from the intermediate pickled files instead than from the original mutation files provided under `--somatic` and `--germline`. If not provided, all the samples passed in `--mutation-sample` will be pickled. Names should match the sample names of the graph/counts files but an equivalence can be set using `--sample-name-map` from mutation parameters.
+
+*Optional subset* parameters: Commands to select a subset of the genes to be processed.
+- `--process-chr`: Default = None. List of chromosomes to be processed. If not provided all chromosomes are processed.
+- `--complexity-cap`: Default = None. Maximum edge complexity of the graph to be processed. If not provided all graphs are processed.
+- `--genes-interest`: Default = None. Genes to be processed. Input is a csv file containing a gencode gene id per line, with no header. The gencode gene id must match the gencode gene ids of the splice graph, which are obtained from version #TODO: check gencode version in SplAdder?. If not provided all genes are processed.
+- `--start-id`: Default = 0. Id of the first gene in the splice graph to be processed. If not provided all genes are processed.
+- `--process-num`: Default = 0. Number of genes to be processed. If provided, the first process-num genes are processed.
+
+*Optional output* parameters: Commands to select output formatting and filtering.
+- `--skip-annotation`: Default = False. Set this parameter to True to skip the generation of a background kmer and peptide files.
+- `--libsize-path`: #TODO: not used in build mode. Remove?
+- `--output-fasta`: Default = False. Set this parameter to True to output the foreground peptides in fasta format. If set to True output number 4 from the [Output files, mode build](#outputs-mode-build) will be generated. If set to False only the kmers will be outputted.
+- `--force-ref-peptides`: Default = False. Set this parameter to True to output mutated peptides even if they are the same as the ones in the reference.
+- `--kmer-database`: Default = None. Absolute path of a file containing kmers in one column, without header. If the kmers contained in this database contain the aminoacid isoleucine (I), it will be converted into leucine (L). A file from uniprot or any other standard library can be provided. The kmers provided in this file will not be outputted if found in the foreground peptides. Please note that is a standard proteome is downloaded from an online resource the proteins should be cut into the kmers length selected under `--kmer`.
```

Comment:
should be --> will not be output (sorry for my grammar)

Reply:
Corrected in last commit